c_api/impl/AuxIndexStructures_c.h

```diff
@@ -126,8 +126,24 @@ int faiss_RangeSearchPartialResult_set_lims(
 int faiss_RangeSearchPartialResult_new_result(
     FaissRangeSearchPartialResult* res, idx_t qno, FaissRangeQueryResult** qr);
 
+
+FAISS_DECLARE_CLASS(DistanceComputer)
```

Comment:
The respective implementations of the functions below appear to be missing. Please follow the pattern seen in the other opaque types to fill in the missing definitions. You can also copy the rest of the documentation to each function where applicable.

Reply:
I just finish the definitions. But I have a trouble with `DistanceComputer::operator()`... I changed the name to `faiss_DistanceComputer_vector_to_query_dis`, but I want to read your suggestions about a better name.

Reply:
Well, I don't have a strong opinion regarding the name of the operator, but it's worth documenting that this function defers to the overloaded function call operator, so that users know where it comes from in the native API.